net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+	]]>
+	</copyright>
+	<name>snmptt</name>
+	<title>Services: SNMPTT</title>
+	<category>Monitoring</category>
+	<include_file>/usr/local/pkg/snmptt.inc</include_file>
+	<addedit_string>SNMPTT has been created/modified.</addedit_string>
+	<delete_string>SNMPTT has been deleted.</delete_string>
+	<menu>
+		<name>SNMP Trap Translator (SNMPTT)</name>
+		<section>Services</section>
+		<url>/pkg_edit.php?xml=snmptt.xml</url>
+	</menu>
+	<service>
+		<name>snmptt</name>
+		<rcfile>snmptt.sh</rcfile>
+		<executable>snmptt</executable>
+		<description>SNMPTT Daemon</description>
+		<custom_php_service_status_command>
+			<![CDATA[
+				""; $output=""; exec("/bin/pgrep -anf snmptt", $output, $retval); $rc=(intval($retval) == 0)
+			]]>
+		</custom_php_service_status_command>
+	</service>
+	<tabs>
+		<tab>
+			<text>General</text>
+			<url>/pkg_edit.php?xml=snmptt.xml</url>
+			<active />
+		</tab>
+		<tab>
+			<text>snmptt.conf</text>
+			<url>/pkg_edit.php?xml=snmptt_conf.xml</url>
+		</tab>
+	</tabs>
+	<fields>
+		<field>
+			<name>General Settings</name>
+			<type>listtopic</type>
+		</field>
+		<field>
+			<fielddescr>Enable</fielddescr>
+			<fieldname>snmpttenabled</fieldname>
+			<description>Enable SNMPTT service.</description>
+			<type>checkbox</type>
+		</field>
+		<field>
+			<fielddescr>SNMPTT System Name</fielddescr>
+			<fieldname>snmptt_system_name</fieldname>
+			<description>
+				<![CDATA[
+					Name of this system for $H variable.  If blank, system name will be the computer's<br/>
+					hostname via Sys::Hostname.<br/>
+				]]>
+			</description>
+			<default_value></default_value>
+			<type>input</type>
+			<size>50</size>
+		</field>
+		<field>
+			<fielddescr>Operation Mode</fielddescr>
+			<fieldname>mode</fieldname>
+			<description>
+				<![CDATA[
+					standalone: snmptt called from snmptrapd.conf<br/>
+					daemon: snmptrapd.conf calls snmptthandle<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>standalone</default_value>
+			<options>
+				<option><name>standalone</name><value>standalone</value></option>
+				<option><name>daemon</name><value>daemon</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Multiple Event</fielddescr>
+			<fieldname>multiple_event</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to allow multiple trap definitions to be executed for the same trap.<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.